src/call-for-proposals.md

```diff
@@ -54,8 +54,12 @@ title: Call for Proposals
 
 <div class="box box_yellow">
   <h3>Young Coders' Events</h3>
-  <p><strong>Help young coders (aimed at ages 8 - 16) learn and make something!</strong></p>
+  <p><strong>Inspire the next generation (aimed at ages 8 - 16) and help shape the future of coding!</strong></p>
   <p>We have Space 4 set up classroom style, with capacity for around 20 young coders (plus their guardians) and a projector and screen.</p>
-  <p>You could share something with the next generation of coders. Help them find something to enjoy, and they can even show it off to the rest of the conference!</p>
-  <p>Sessions are 60 minutes long and are not recorded.</p>
+  <p>Are you a developer, educator, or just love tech? Guide young minds in an exciting hands-on session - whether it’s coding a simple game, bringing robots to life, creating an interactive website or hacking on fun puzzles and challenges.</p>
+  <p><strong>Let them shine!</strong></p>
+  <p>
+  At the end, participants can show off their creations to the rest of the conference—because every coder deserves a moment in the spotlight!
```

Comment:
It won't break anything, but there's also no need for the line wrapping here. No urgent need to fix it though if you don't think it's worth it.